zos_concepts/volume_management/volume_initialization/init_dasd_vol_and_run_sample_jcl/files/HELLO.jcl

```diff
@@ -0,0 +1,32 @@
+//******************************************************************************
+//* Copyright (c) IBM Corporation 2023
+//******************************************************************************
+//******************************************************************************
+//* Configure the job card as needed, most common keyword parameters often
+//* needing editing are:
+//* CLASS: Used to achieve a balance between different types of jobs and avoid
+//*        contention between jobs that use the same resources.
+//* MSGLEVEL: controls hpw the allocation messages and termination messages are
+//*           printed in the job's output listing (SYSOUT).
+//* MSGCLASS: assign an output class for your output listing (SYSOUT)
+//*
+//* Notes:
+//*   If you change the job name, you must update the playbook job name
+//*   references.
+//******************************************************************************
+//HELLO    JOB (T043JM,JM00,1,0,0,0),'HELLO WORLD - JRM',
+//             MSGCLASS=X,MSGLEVEL=1,NOTIFY=&SYSUID
+//*
+//* PRINT "HELLO WORLD" ON JOB OUTPUT
+//*
+//* NOTE THAT THE EXCLAMATION POINT IS INVALID EBCDIC FOR JCL
```

Comment:
This is an irrelevant old comment probably when doing some negative testing. Probably don't need that. 
//* NOTE THAT THE EXCLAMATION POINT IS INVALID EBCDIC FOR JCL
AND WILL CAUSE A JCL ERROR